packages/authentication/src/index.ts

```diff
@@ -3,6 +3,8 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+// Keep backward compatibility
+export {UserProfile} from '@loopback/security';
```

Comment:
@jannyHou you should remove the export if you want a semver major :-)

Reply:
@derdeka reverted :)